lib/generators/sufia/templates/migrations/change_featured_work_generic_file_id_to_generic_work_id.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFeaturedWorkGenericFileIdToGenericWorkId < ActiveRecord::Migration
+  def change
+    return unless column_exists?(:featured_works, :generic_file_id)
+    rename_column :featured_works, :generic_file_id, :generic_work_id
```

Comment:
I don't see where you're finding the `generic_file_id` column.  It was created as `generic_work_id` here: https://github.com/projecthydra/sufia/blob/master/lib/generators/sufia/templates/migrations/create_featured_works.rb#L5